dev/io.openliberty.org.apache.myfaces.4.0_fat/test-applications/AnnotationLiteralsTest.war/src/io/openliberty/org/apache/faces40/fat/literals/TestBean.java

```diff
@@ -0,0 +1,273 @@
+        Map<String, String> myStringMap = CDI.current().select(new TypeLiteral<Map<String, String>>() {}, ManagedProperty.Literal.of("#{managedPropertyBean.myStringMap}")).get();
+
+        return Boolean.toString(myInteger.intValue() == 42 && myStringMap.containsKey("foo") && myStringMap.get("foo").equals("bar"));
+    }
+
+    /**
+     * TESTCOOKIE - set in request
+     *
+     * @see AnnotationLiteralsTest#testRequestCookieMap()
+     *
+     * @return the requestCookieMap
+     */
+    public String getRequestCookieMap() {
+        Map<String, Object> requestCookieMap = CDI.current().select(new TypeLiteral<Map<String, Object>>() {}, RequestCookieMap.Literal.INSTANCE).get();
+        return ((Cookie) requestCookieMap.get("TESTCOOKIE")).getValue();
+    }
+
+    /**
+     * fooAttribute - set in AttributeFilter class
+     *
+     * @see AttributeFilter
+     *
+     * @return the requestMap
+     */
+    public String getRequestMap() {
+        Map<String, Object> requestMap = CDI.current().select(new TypeLiteral<Map<String, Object>>() {}, RequestMap.Literal.INSTANCE).get();
+        return (String) requestMap.get("fooAttribute");
+    }
+
+    /**
+     * foo - set as URL param
+     *
+     * @see AnnotationLiteralsTest#testRequestParameterMap()
+     *
+     * @return the requestParameterMap
+     */
+    public String getRequestParameterMap() {
+        Map<String, String> requestParameterMap = CDI.current().select(new TypeLiteral<Map<String, String>>() {}, RequestParameterMap.Literal.INSTANCE).get();
+        return requestParameterMap.get("foo");
+    }
+
+    /**
+     * foo - set as URL params
+     *
+     * @see AnnotationLiteralsTest#testRequestParameterValuesMap()
+     *
+     * @return the requestParameterValuesMap
+     */
+    public String getRequestParameterValuesMap() {
+        Map<String, String[]> requestParameterValuesMap = CDI.current().select(new TypeLiteral<Map<String, String[]>>() {}, RequestParameterValuesMap.Literal.INSTANCE).get();
+        return Stream.of(requestParameterValuesMap.get("foo")).collect(Collectors.joining(":"));
+    }
+
+    /**
+     * /fake.xhtml - path to view defined in FaceletViewBean class
+     *
+     * @see FaceletViewBean
+     *
+     * @return the view
+     */
+    public String getView() {
+        Facelet view = CDI.current().select(new TypeLiteral<Facelet>() {}, View.Literal.of("/fake.xhtml")).get();
+        return Boolean.toString(view != null && view.getClass().getCanonicalName().contains(FaceletViewBean.class.getCanonicalName()));
+    }
+
+    /**
+     * Test data is the session itself. Every request to a servlet is within a session.
+     *
+     * @see HttpSession
+     *
+     * @return the sessionMap
+     */
+    public String getSessionMap() {
+        Map<String, Object> sessions = CDI.current().select(new TypeLiteral<Map<String, Object>>() {}, SessionMap.Literal.INSTANCE).get();
+        return Boolean.toString(sessions != null && !sessions.isEmpty());
+    }
+}
```

Comment:
New line, but @pnicolucci could address this later when we re-enable the testFacesConfig

Reply:
yea, I'll add it to the todo list.